collectors/python.d.plugin/tor/README.md

```diff
@@ -0,0 +1,48 @@
+# tor
+
+Module connects to tor control port to collect traffic statistics.
+
+**Requirements:**
+* `tor` program
+* `stem` python package
+
+It produces only one chart:
+
+1. **Traffic**
+ * read
+ * write
+
+### configuration
+
+Needs `control_port` and `password`
```

Comment:
`password` isn't required; tor doesn't require a password be used to access the control port.